lib/fpm/package/srpm.rb

```diff
@@ -0,0 +1,243 @@
+require "fpm/namespace"
+require "fpm/package"
+require "fpm/errors"
+require "fpm/util"
+
+# For handling conversion
+require "fpm/package/cpan"
+
+class FPM::Package::SRPM < FPM::Package
```

Comment:
Lots of this file is copied directly from `rpm.rb` (the FPM::Package::RPM class).  I'd like to reduce code duplication if we can, such as the duplicate methods for architecture, to_s, etc, and the nearly identical `output` method.